pkg/proxy/headers/forwarding_test.go

```diff
@@ -46,7 +46,7 @@ var testHops1 = &Hop{
 var testHops2 = &Hop{
 	RemoteAddr: "5.6.7.8",
 	Scheme:     "http",
-	Host:       "foo.com",
+	Host:       "fbo.com",
```

Comment:
Heh

Reply:
lol search / replace mishap artifact. i'll restore that one pre-merge